README.md

```diff
@@ -15,8 +15,76 @@ npm install glsl-sandbox --save
 ```
 
 ## Usage
+```js
+import { WebGLRenderer, PerspectiveCamera, Vector3 } from 'three';
+import { resolveLygia } from 'resolve-lygia';
+import { GlslSandbox } from 'glsl-sandbox';
 
-TODO
+const renderer = new WebGLRenderer();
+const camera = new PerspectiveCamera(45, window.innerWidth / window.innerHeight, 0.01, 100);
+
+const fragmentShader = resolveLygia(`
+#define PLATFORM_WEBGL
+
+uniform sampler2D   u_scene;
+
+uniform vec2        u_resolution;
+
+varying vec2        v_texcoord;
+
+#include "lygia/distort/chromaAB.glsl"
+
+void main() {
+    vec4 color = vec4(vec3(0.0), 1.0);
+    vec2 pixel = 1.0 / u_resolution;
+    vec2 st = gl_FragCoord.xy * pixel;
+    vec2 uv = v_texcoord;
+
+#if defined(BACKGROUND)
+    // Optional fullscreen quad rendered behind the 3D scene
+    // It will color the background in red
+    color.r = 1.0;
+
+#elif defined(POSTPROCESSING)
+    // Postprocessing pass, applies chromatic aberration
+    // and displays the scene directly to the screen
+    color = chromaAB(u_scene, st);
+
+#else
+    // Main shade if it's rendered as a 2D scene
+    // and the shape's surface when rendered a 3D scene 
+    color = texture2D(u_scene, uv);
```

Comment:
Mmm... in the else is where `u_scene` is render inside the buffer... you can not actually read from it here. 
This is where the geometry is

Reply:
Apologies for that. Just pushed some changes, does that work for you?